dash/development/base_component.py

```diff
@@ -22,7 +22,15 @@
         eg: html.A(href=os.getenv('DASH_LOGOUT_URL'))
     """
         )
-    }
+    },
+    "dash_table": {
+        "DataTable": textwrap.dedent(
+            """
+        The dash_table.DataTable will be removed from the builtin dash components in Dash 4.0
```

Comment:
This should be Dash 5.0 now?

Reply:
I hope so - not much of a notice period if Dash 4 is released soon.